doc/move_group_interface/src/move_group_interface_tutorial.cpp

```diff
@@ -360,38 +360,75 @@ int main(int argc, char** argv)
   visual_tools.trigger();
   visual_tools.prompt("next step");
 
-  // Now, let's attach the collision object to the robot.
+  // Attaching Objects
+  // ^^^^^^^^^^^^^^^^^
+  //
+  // Now, let's attach a different type of collision object to the robot, for the purposes of "manipulating" it.
```

Comment:
```suggestion
  // You can attach objects to the robot, so that it moves with the robot geometry. This simulates picking up the object for the purpose of manipulating it. The motion planning will avoid collisions between the two objects as well.
```